test/operations-test.js

```diff
@@ -14,7 +14,7 @@ describe(__filename + '#', function () {
     [false, [false,'false', true], [false]],
     [true, [1, true], [true]],
     [0, [0,'0'], [0]],
-    [null, [void 0, null], [null]],
```

Comment:
This is quite controversial, but i failed to find a non-breaking workaround. That's how would mongo work though. 
```
db.tests.find({a: null})
db.tests.insert({b: 1})
db.tests.insert({a: undefined, b: 2})

db.tests.find({})
{ "_id" : ObjectId("563d00410b07e633a7c83542"), "a" : null }
{ "_id" : ObjectId("563d01360b07e633a7c83543"), "b" : 1 }
{ "_id" : ObjectId("563d01560b07e633a7c83544"), "a" : null, "b" : 2 }

db.tests.find({a: null})
{ "_id" : ObjectId("563d00410b07e633a7c83542"), "a" : null }
{ "_id" : ObjectId("563d01360b07e633a7c83543"), "b" : 1 }
{ "_id" : ObjectId("563d01560b07e633a7c83544"), "a" : null, "b" : 2 }

db.tests.find({a: undefined})
Error: error: {
	"$err" : "Can't canonicalize query: BadValue cannot compare to undefined",
	"code" : 17287
}
```